USAGE.md

```diff
@@ -4894,7 +4894,7 @@ puts response.headers
 **This endpoint allows you to retrieve the associated link whitelabel for a subuser.**
 
 Link whitelables can be associated with subusers from the parent account. This functionality allows
-subusers to send mail using their parent's linke whitelabels. To associate a link whitelabel, the parent account
+subusers to send mail using their parent's like whitelabels. To associate a link whitelabel, the parent account
```

Comment:
Can you please change this to "Link whitelabels" ?

Reply:
Done!
Thanks, @mbernier!